btk/draw_blends.py

```diff
@@ -274,6 +278,20 @@ def __next__(self):
                 blend_images[s.name][i] = batch_results[i][0]
                 isolated_images[s.name][i] = batch_results[i][1]
                 batch_blend_cat[s.name].append(batch_results[i][2])
+
+            if self.save_path is not None:
+                if not os.path.exists(os.path.join(self.save_path, s.name)):
+                    os.mkdir(os.path.join(self.save_path, s.name))
```

Comment:
This makes sense to me, so now `save_path` is a directory which will contain one folder for each survey containing the images for that survey.